rusk/src/lib/chain/vm.rs

```diff
@@ -120,6 +122,17 @@ impl VMExecution for Rusk {
                 anyhow::anyhow!("Cannot finalize txs: {inner}!!")
             })?;
 
+        let current_commit = self.state_root();
+        let r = Migration::migrate(
```

Comment:
Migration must be done in all state transition calls (EST/VST/Accept/Finalize) otherwise the state root will be different from the one in the block

Reply:
done